### ISSUE TITLE
CDK k8s Vagrantfile needs to disable default openshift setup

### ISSUE THREAD

**Issue by navidshaikh** [projectatomic/adb-atomic-developer-bundle#334]
Issue and its explanation is at https://github.com/projectatomic/vagrant-service-manager/issues/162

TLDR:
- VSM plugin can only support to configure `docker` and `openshift` services via Vagrantfile. 
- For `kubernetes` services, one has to explicitly enable it via shell provisioning block
- starting openshift service for CDK box is a default setup in VSM plugin
- Case where user configures `kubernetes` via shell provisioning for CDK box, it conflicts with default setup of `openshift` and `kubernetes` service is not started properly, rather `openshift` service is started

Fix:
- Disable `openshift` service in VSM config in k8s Vagrantfile
- Ideal fix: Have support for configuring `kubernetes` service via the VSM plugin and remove the need of shell provisioning block altogether for kubernetes

**Comment by brgnepal** [comment]
> Disable openshift service in VSM config in k8s Vagrantfile

Instead of this, I would prefer to re-word as :

```
Specify only 'docker' service in VSM config in k8s Vagrantfile as 
     config.servicemanager.services = 'docker'
```
